libcontainer/rootfs_linux_test.go

```diff
@@ -52,7 +51,41 @@ func TestCheckBindMountOnProc(t *testing.T) {
 	dest := "/rootfs/proc/"
 	err := checkProcMount("/rootfs", dest, m)
 	if err != nil {
-		t.Fatalf("bind-mount of procfs on top of /proc should not return an error: %v", err)
+		t.Fatalf("bind-mount of procfs on top of /proc should not return an error (for now): %v", err)
+	}
+}
+
+func TestCheckTrickyMountOnProc(t *testing.T) {
```

Comment:
These two seem c&p and table tests might be a better way to avoid the duplication here. It can be on a follow-up PR if you prefer that too.

Reply:
Yeah, I just wanted to keep it as-is for now.